### ISSUE TITLE
quick navigation key to jump to the next or previous tab in a web site or application

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3956]
**Reported by heikofolkerts on 2014-03-06 16:24**
In web applications the information is often devided in tabs the user must activate to choose the desired information. NVDA should be able to navigate between those tabs like navigating to headings, lists etc. I would propose the p key for that - p like tab Page.

The enhancement will be most useful in web applicatins that get more and more popular.

**Comment by bhavyashah** [comment]
I tend to agree with @jcsteh's reasoning in https://github.com/nvaccess/nvda/issues/3956#issuecomment-155318048 against the allotment of another single letter navigation key to tabs. Unfortunately, we can no longer contact the original author of this ticket and he/she did not follow-up or respond to Jamie's arguments either. Thus, I suggest closing. @ehollig

**Comment by ehollig** [comment]
Closing because we are unable to contact the original poster for additional information.
